api/external/applications/applications.proto

```diff
@@ -87,6 +87,27 @@ service ApplicationsService {
     option (chef.automate.api.iam.policy).action = "applications:serviceGroups:delete";
   };
 
+  rpc GetDisconnectedServicesConfig(GetDisconnectedServicesConfigReq) returns (GetDisconnectedServicesConfigRes) {
+    option (google.api.http).get = "/beta/retention/service_groups/disconnected_services/config";
+    // TODO: need new resources/policies for this
+    option (chef.automate.api.policy).resource = "service_groups";
```

Comment:
Is it service groups or is there like an config resource?

Reply:
This is what ingest does:
```
    option (chef.automate.api.policy).resource = "cfgmgmt:scheduler:missing-nodes";
    option (chef.automate.api.policy).action = "configure";
    option (chef.automate.api.iam.policy).resource = "retention:nodes";
    option (chef.automate.api.iam.policy).action = "retention:nodes:update";
```

I think we'd do something similar. Tracking this in #1371